### PR TITLE
Fix tile to file

### PR DESCRIPTION
**Pull request by DMotorygin**
JIRA TICKET: RPRMAYA-1093

PURPOSE: Make tile render correctly use save to file feature

EFFECT FOR THE USER: User can now see AOVs for full image rendered via tile rendering saved in file, just like with regular render.

NOTE: Thought its related to Hybrid RC5 but turned out to be other issue entirely

### CODE REVIEW

**Comment by DavidSpeicher** [comment]
approved

File: FireRender.Maya.Src/FireRenderCmd.cpp [hunk 1]
```diff
@@ -233,15 +234,56 @@ MStatus FireRenderCmd::renderFrame(const MArgDatabase& argData)
...
-			MString cmd;
+			AOVPixelBuffers& outBuffers = s_production->GetContext()->PixelBuffers();
+			for (const auto& colorPixelBuffer : outBuffers)
+			{
+				unsigned aov_id = colorPixelBuffer.first;
+
+				// get correct path
+				MString folder = aovs->getAOV(aov_id)->folder;
+
+				//// - replace '\' with '/'
```
Comment (Vsevolod1983) [inline FireRender.Maya.Src/FireRenderCmd.cpp:247]: do we need these comments ?

File: FireRender.Maya.Src/FireRenderProduction.cpp [hunk 2]
```diff
@@ -1095,7 +1095,7 @@ void FireRenderProduction::RenderTiles()
 		RenderViewUpdater::UpdateAndRefreshRegion(data, m_width, m_height, RenderRegion(0, m_width - 1, m_height - 1, 0));
 	});
 
-	outBuffers.clear();
+	//outBuffers.clear();
```
Comment (Vsevolod1983) [inline FireRender.Maya.Src/FireRenderProduction.cpp:1098]: do we need this comment ?

**Review by Vsevolod1983 — CHANGES_REQUESTED** [review]
remove some comments pls if not needed

**Review by Vsevolod1983 — APPROVED** [review]

**Pull request merged**